src/rt/dwarfeh.d

```diff
@@ -765,3 +774,10 @@ enum LsdaResult
     cleanup,    // cleanup found (i.e. finally or destructor)
     handler,    // handler found (i.e. a catch)
 }
+
+void terminate(uint line) @nogc
+{
+    static char* p;
+    printf("dwarfeh(%u) fatal error\n", line);
+    *p = 0;     // generate seg fault
```

Comment:
It might be better to use `_Exit` instead, like [here](https://github.com/D-Programming-Language/druntime/blob/59ff0a44c98ffe90b94aee36df9485032e56a915/src/rt/sections_elf_shared.d#L882), instead of spamming some user's system with core dumps.